### ISSUE TITLE
How to POST (lng, lat) to Firebase as GeoFire ?

### ISSUE THREAD

**Issue by HTAD** [firebase/geofire#9]
Hello,

I have a list of `lng` and `lat` data that will be post to Firebase. I use Restful API for posting. However I don't know how to post them as GeoFire.

Hope someone can help me !

Thanks !

Best regards,

**Comment by netdawgx** [comment]
This is platform non-specific:
Assuming your lat/long data is stored in a database you can create a function that connects to the database, retrieves the coordinates and then uses the GeoFire methods to create "GeoFire data" that will post to your Firebase database. This would be recursive for as many coordinates as you have.

Unless you are asking about how to install or use GeoFire? Are you familiar with how to connect to/store/retrieve data from a Firebase database?

**Comment by jwngr** [comment]
If I understand your question correctly, you want to add data to a GeoFire index directly via the REST API. This is possible, but not really a use case that we've put a lot of thought into. All the GeoFire SDKs use the same data model, so you can reverse engineer it ([look at some rules](https://github.com/firebase/geofire-js/blob/master/examples/securityRules/rules.json) to see the expected format) and write in that format via the REST API. However, you'll have to do all the geohashing yourself (which is [non-trivial](https://github.com/firebase/geofire-js/tree/master/src)) this data model may change at some point so you'd have to handle that case.

So, in short, you *can* use the REST API to write a GeoFire index, but it is not going to be an easy task by any means.
